.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/app.log
```

Comment:
The path `logs/app.log` should be adjusted to `logs/` to ignore all files within the `logs` directory, not just `app.log`. This ensures that any other log files generated in the `logs` directory are also ignored.